apps/condo/domains/user/schema/SendMessageToSupportService.js

```diff
@@ -19,13 +20,12 @@ const { sendMessage } = require('@condo/domains/notification/utils/serverSchema'
 const { Resident, ServiceConsumer } = require('@condo/domains/resident/utils/serverSchema')
 const access = require('@condo/domains/user/access/SendMessageToSupportService')
 
-const EMAIL_NOT_FOUND_ERROR = 'support_email_mobile variable not found. help_requisites must be specified in .env'
-const scriptLifecycleName = process.env.npm_lifecycle_event
+const logger = getLogger('SendMessageToSupportService')
```

Comment:
Where this logger is used in this module?

Reply:
26 line